site/webpack.mix.js

```diff
@@ -0,0 +1,51 @@
+let mix = require('laravel-mix');
+let argv = require('yargs').argv;
+let bin = require('./tasks/bin');
+let WebpackShellPlugin = require('webpack-shell-plugin');
+
+let env = argv.e || argv.env || 'local';
+var port = argv.p || argv.port || 3000;
```

Comment:
Actually, my two cents, I'd use `const`, since these variables (mix, argv, binc, WebpackShellPlugin...) are not meant to change, and since there's a debate about the proper use of let and const, I'd say this is the most rational approach:

https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

> `const` is a signal that the identifier won’t be reassigned.
`let` is a signal that the variable may be reassigned, such as a counter in a loop, or a value swap in an algorithm.

If this has already been discussed and I'm just opening the Pandora Box again... ignore me :)

Thanks!